consensus/ibft/verifier.go

```diff
@@ -184,3 +182,43 @@ func (i *backendIBFT) extractProposer(header *types.Header) (types.Address, erro
 
 	return proposer, nil
 }
+
+func (i *backendIBFT) getProposalHashFromBlockBytes(ethereumBlock []byte, round *uint64) (types.Hash, error) {
```

Comment:
Attached document about compatibility of this change with checkpointing/slashing for 0.7.x. https://www.notion.so/polygontechnology/Signing-in-goibft-6a3ba4a50deb4efd85c1e2d71f401a6d

Reply:
As discussed yesterday, the hash for Committed Seals is calculated by keccak256(block hash + round bytes) from commit c0d1d00f0dbfd7da255fa6b7756013596660970b